Documentation/UserRoundTrip.rst

```diff
@@ -0,0 +1,345 @@
+.. _user-round-trip-typo3-extension-type:
+
+TYPO3 extension
+---------------
+
+.. uml::
+
+   !include <tupadr3/font-awesome-5/typo3>
+
+   skinparam DefaultTextAlignment center
+   skinparam RectangleBackgroundColor transparent
+   skinparam RectangleBorderColor transparent
+   skinparam UsecaseFontColor<< inactive >> #b9b9b9
+   skinparam UsecaseBorderColor<< inactive >> #b9b9b9
+   skinparam UsecaseBackgroundColor<< inactive >> #FEFEFE
+
+   rectangle "<color:#f49700><$typo3></color>\n TYPO3 extension" as typo3actor
+   usecase "Packagist" as packagist
+   usecase "VCS repository" as vcsRepository
+   usecase "Documentation" as documentation
+   usecase "TYPO3 Extension Repository" as ter
+
+   typo3actor -left-> packagist
+   typo3actor -right-> vcsRepository
+   typo3actor -up-> documentation
+   typo3actor -down-> ter
+
+   packagist <-up-> documentation
+   documentation <-down-> vcsRepository
+   vcsRepository <-down-> ter
+   ter <-up-> packagist
+
+   hide << inactive >> stereotype
+
+Guide the user to all four aspects of the TYPO3 extension, which is usually
+hosted on both TER and Packagist, by configuring:
+
+#. :ref:`user-round-trip-documentation-aspect`
+#. :ref:`user-round-trip-vcs-repository-aspect`
+#. :ref:`user-round-trip-ter-aspect`
+#. :ref:`user-round-trip-packagist-aspect`
+
+For example, look at the *Apache Solr* extension and try navigating all four
+aspects by simply clicking on a redirect link on each page - starting at the
+`Apache Solr TER page <https://extensions.typo3.org/extension/solr>`__.
+
+
+.. _user-round-trip-composer-package-type:
+
+Composer package
+----------------
+
+.. uml::
+
+   !include <tupadr3/font-awesome-5/typo3>
+
+   skinparam DefaultTextAlignment center
+   skinparam RectangleBackgroundColor transparent
+   skinparam RectangleBorderColor transparent
+   skinparam UsecaseFontColor<< inactive >> #b9b9b9
+   skinparam UsecaseBorderColor<< inactive >> #b9b9b9
+   skinparam UsecaseBackgroundColor<< inactive >> #FEFEFE
+
+   rectangle "<color:#f49700><$typo3></color>\n Composer package" as typo3actor
+   usecase "Packagist" as packagist
+   usecase "VCS repository" as vcsRepository
+   usecase "Documentation" as documentation
+   usecase "TYPO3 Extension Repository" << inactive >> as ter
+
+   typo3actor -left-> packagist
+   typo3actor -right-> vcsRepository
+   typo3actor -up-> documentation
+   typo3actor -[hidden]down-> ter
+
+   packagist <-up-> documentation
+   documentation <-down-> vcsRepository
+   vcsRepository <-[hidden]down-> ter
+   ter <-[hidden]up-> packagist
+
+   hide << inactive >> stereotype
+
+A Composer package that is not a TYPO3 extension is usually hosted only on
+Packagist. Therefore, guide the user to three aspects of the package by
+configuring:
+
+#. :ref:`user-round-trip-documentation-aspect`
+#. :ref:`user-round-trip-vcs-repository-aspect`
+#. :ref:`user-round-trip-packagist-aspect`
+
+For example, examine the *TYPO3 Console* package and try navigating through its
```

Comment:
Let us try to make shorter sentences here

Reply:
I think the balance of short and long sentences is fine in this article.